cicd/bin/check_junit_files

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import logging
+import sys
+
+from glob import iglob
+from junitparser import JUnitXml
+
+if __name__ == "__main__":
+
+    def errors_or_failures_found(junit_report):
```

Comment:
You could just define this up above outside of this scope

Reply:
yeah I forgot to move it :D - done. thanks!